### PR TITLE
Resolves #51 once approved and reviewed - adding images to game info, details, wishlist.

### LINKED ISSUE #51: Images for game list/details view
As a user, i want to see an image of the game I selected on the list and details view. It will help make the site more enjoyable to look at.

### Acceptance criteria:
 - Each game should have an image of the game to display (pull them from BoardGameGeek.com)

### Tech notes:
 - Update the game table migration to include an "image_url" text column
 - Update the game seeder to include an image_url pulled from BoardGameGeek.com
 - Update the game list and details views to display the image at an appropriate size (obviously there will be more room on the details view.

### PR DESCRIPTION
**Pull request by JessEpley-dev**
Resolves issue #51 once reviewed and approved.

Added images to game details, game list, and wishlist. Added some tiny styling adjustments to how details are displayed.

You will need to run migrations / seed database with new seeder info and game table: `sail artisan migrate:fresh --seed`

### CODE REVIEW

File: app/Models/Game.php [hunk 1]
```diff
@@ -18,7 +18,8 @@ class Game extends Model
         'release_year',
         'playing_time_minutes',
         'min_number_players',
-        'max_number_playeres'];
+        'max_number_playeres',
```
Comment (12bitsample) [inline app/Models/Game.php:21]: Maybe there is a reason it is like this, but I notice that "players" seems to be misspelled here!

**Review by mdigrandi — DISMISSED** [review]
Nothing is standing out to me as wrong on this, are you having trouble with any sections? Great work!

File: app/Models/Game.php (hunk 1 above)
Comment (JessEpley-dev) [reply app/Models/Game.php:21]: Nope, this is just a typo. Good eyes :)

**Review by lrapp1 — APPROVED** [review]
Great job Jessica! Images are fun and make sense with the product, user is able to see images/game-info regardless of logged in/out, all tests pass! Merging this bad boiiii

**Pull request merged**